deploy/config-spec/properties.yaml

```
@@ -14,3 +14,22 @@ properties:
           required: true
       asOfVersion: "0.0.0"
       description: "The secret where the Superset credentials are stored."
+  - property: &rowLimit
+      propertyNames:
+        - name: "ROW_LIMIT"
+          kind:
+            type: "file"
+            file: "superset_config.py"
+      datatype:
+        type: "integer"
+      defaultValues:
+        - fromVersion: "0.0.0"
+          value: "50000"
+      recommendedValues:
+        - fromVersion: "0.0.0"
+          value: "50000"
+      roles:
+        - name: "node"
+          required: false
+      asOfVersion: "0.0.0"
+      description: "row limit when requesting chart data"
```

```suggestion
      description: "row limit when reading data from a data source"
```
It is also used for SQL Lab I guess

---

I took the description from the upstream superset configuration (https://github.com/apache/superset/blob/1.4.2/superset/config.py#L125-L126):

```python
# default row limit when requesting chart data
ROW_LIMIT = 50000
```

I think it is easier to just copy and paste their comments instead of reinventing them.

---

Didn't know that, good point